p5js/FeatureExtractor_Image_Classification/sketch.js

```diff
@@ -24,21 +25,22 @@ function setup() {
   // Extract the already learned features from MobileNet
   featureExtractor = ml5.featureExtractor('MobileNet', modelReady);
   // Create a new classifier using those features and give the video we want to use
-  classifier = featureExtractor.classification(video);
+  classifier = featureExtractor.classification(video, videoReady);
```

Comment:
Is there a reason why we have the `featureExtractor` tell us when the video is ready rather than the `p5.MediaElement` `video`?

Reply:
Feature extractor creates a custom video element with the right size for mobilenet. This is an async process. Giving a callback allows to use it like this. If not, it will create a Promise